internal/description/description.go

```diff
@@ -53,6 +53,7 @@ A user’s roles apply to all the clusters in the project.`
 	ListDBUsers          = "List Atlas database users for a project."
 	ListEvents           = "List events for an organization or project"
 	UpdateDBUser         = "Update a MongoDB dbuser in Atlas."
+	ProcessMeasurements  = "Get process measurements."
```

Comment:
I am happy to change these descriptions

Reply:
what about "Get measurements for a given host" ?